opal/group.go

```diff
@@ -563,7 +562,27 @@ func resourceGroupRead(ctx context.Context, d *schema.ResourceData, m any) diag.
 	}
 	d.Set("reviewer", reviewers)
 
-	// XXX: Read out message channels.
```

Comment:
why dropped?

Reply:
I think we actually already implemented message channels, if you see above, we do parse `audit_message_channel(s)`, which is the only option for groups